appveyor/build_task.bat

```diff
@@ -51,7 +51,7 @@ if %errorlevel% neq 0 exit /b 3
 if "%THREAD_SAFE%" equ "0" set ADD_CONF=%ADD_CONF% --disable-zts
 if "%INTRINSICS%" neq "" set ADD_CONF=%ADD_CONF% --enable-native-intrinsics=%INTRINSICS%
 
-set EXT_EXCLUDE_FROM_TEST=snmp,oci8_12c,pdo_oci,pdo_firebird,ldap,imap,ftp
+set EXT_EXCLUDE_FROM_TEST=bz2,exif,fileinfo,ffi,ftp,gmp,imap,ldap,oci8_12c,pdo_firebird,pdo_oci,snmp,soap,sodium,sqlite3,tidy
```

Comment:
Would it be possible to convert this into a whitelist rather than a blacklist? I think that will make more sense going forward.

Reply:
This value is passed to the config option `--with-test-ini-ext-exclude` where blacklisting generally makes sense (it's only used if `--enable-snapshot-build` is also given, and usually you want to work with all available extensions).

For the test runner on AppVeyor, whitelisting would make more sense in the long run; would be trivial to change if it wasn't batch sctipting. I'll try to have a look.

Reply:
Whitelist would be nice going forward, but I think we can land this as-is. Maybe add a comment that (most of) these are actually tested, just loaded via `--EXTENSIONS--`.